subsys/usb/device_next/class/usbd_cdc_acm.c

```diff
@@ -1102,6 +1127,9 @@ static struct usbd_cdc_acm_desc cdc_acm_desc_##n = {				\
 		     " is not assigned to a USB device controller");		\
 										\
 	CDC_ACM_DEFINE_DESCRIPTOR(n);						\
+	USBD_DESC_STRING_DEFINE(iad_desc_nd_##n,				\
+				DT_NODE_FULL_NAME(DT_DRV_INST(n)),		\
```

Comment:
I don't really like using node name as a string descriptor. The identifiers used in code are frequently not what the end users should see.

USB provides multiple language string descriptor support. The fact that in practice only a single locale tends to be used in many devices is separate concern. However, we should really have some framework for managing string descriptors. Deriving string descriptor from node label is pretty bad solution in my opinion.

Reply:
Being only way to accurately identify the interface does not make it good for string descriptor.